### DISCUSSION TITLE
BlitzWolf 400ml Aroma Diffuser not using ESP but WBR3

### DISCUSSION THREAD

**Discussion by oangelo** [arendst/Tasmota#13001] [Q&A] [answered]
I have bought a [BlitzWolf 400ml Aroma Diffuser](https://templates.blakadder.com/blitzwolf_BW-FUN3.html) and I wish I could flash Tasmota on it, but my version did not came with an ESP, but with an [WBR3](https://developer.tuya.com/en/docs/iot/wbr3-module-datasheet?id=K9dujs2k5nriy). I wish to replace WBR3, but I do not know which ESP version will be better. Someone can help me? Thanks.
![WhatsApp Image 2021-08-27 at 19 30 13 (1)](https://user-images.githubusercontent.com/1669411/131194866-26aedef2-f0d7-492e-961b-e7d1b37b9895.jpeg)
![WhatsApp Image 2021-08-27 at 19 30 13](https://user-images.githubusercontent.com/1669411/131194875-ddb16188-c293-4bf1-bfb0-e0b09bf970c2.jpeg)

**Comment by sfromis** [comment, marked as answer]
ESP-12 should be a good replacement. [Pinout of WBR3](https://developer.tuya.com/en/docs/iot/wbr3-module-datasheet) looks very similar.
